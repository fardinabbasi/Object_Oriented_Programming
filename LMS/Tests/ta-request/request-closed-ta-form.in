POST login ? id 0 password UT_account
POST course_offer ? course_id 1 professor_id 810201475 capacity 60 time Saturday:14-16 exam_date 1403/4/3 class_number 5
POST course_offer ? course_id 7 professor_id 810201475 capacity 80 time Monday:13-15 exam_date 1403/4/6 class_number 7
POST logout ?
POST login ? id 810201475 password ry
POST post ? title "start" message "hey"
POST ta_form ? course_id 1 message "need TA for AP"
POST ta_form ? course_id 2 message "need TA for compiler"
POST logout ?
POST login ? id 810101551 password hehe
GET personal_page ? id 810201475
POST ta_request ? professor_id 810201475 form_id 2
POST logout ?
POST login ? id 810101456 password blabla
POST ta_request ? professor_id 810201475 form_id 3
POST logout ?
POST login ? id 810201475 password ry
POST close_ta_form ? id 2
accept
POST close_ta_form ? id 1
POST logout ?
POST login ? id 810101551 password hehe
GET personal_page ? id 810201475
GET post ? id 810201475 post_id 2
GET notification ?
POST logout ?
POST login ? id 810201475 password ry
POST close_ta_form ? id 3
reject
POST logout ?
POST login ? id 810101456 password blabla
POST ta_request ? professor_id 810201475 form_id 2
GET notification ?
POST logout ?