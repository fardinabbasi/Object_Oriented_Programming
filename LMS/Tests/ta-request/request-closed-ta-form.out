OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
khousravi Computer Engineering Professor Advanced Programming,compiler
3 TA form for compiler course
2 TA form for Advanced Programming course
1 "start"
OK
OK
OK
OK
OK
OK
We have received 1 requests for the teaching assistant position
810101551 parisa 4: 
Not Found
OK
OK
khousravi Computer Engineering Professor Advanced Programming,compiler
3 TA form for compiler course
1 "start"
Not Found
1 Advanced Programming: Your request to be a teaching assistant has been accepted.
810201475 khousravi: New Course Offering
810201475 khousravi: New Course Offering
OK
OK
We have received 1 requests for the teaching assistant position
810101456 ali 7: 
OK
OK
Not Found
2 compiler: Your request to be a teaching assistant has been rejected.
810201475 khousravi: New Course Offering
810201475 khousravi: New Course Offering
OK
